cmd/server.go

```diff
@@ -693,3 +697,13 @@ func (s *ServerCmd) withErrPrint(f func(*cobra.Command, []string) error) func(*c
 func (s *ServerCmd) printErr(err error) {
 	fmt.Fprintf(os.Stderr, "%sError: %s%s\n", "\033[31m", err.Error(), "\033[39m")
 }
+
+func isValidLogLevel(level string) bool {
+	for _, logLevel := range ValidLogLevels {
+		if strings.EqualFold(logLevel, level) {
```

Comment:
this should just be `level == logLevel`. We don't need to do a case insensitive compare here since it's already downcased.

Reply:
I was on the fence about that exact thing. I decided to leave it so that in the event that someone called isValidLogLevel() elsewhere, the comparison would still work. I'm not partial to either, so I'll make that change.

Reply:
ahh okay. But technically it's not a valid log level right?